### ISSUE TITLE
Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++.

### ISSUE THREAD

**Issue by raotolehitesh** [BWMuller/background_location#3]
Library working fine.
I get location on Application Active and Resume state.
When I terminate application then location fetching is stopped.
Its show 
Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: com.almoullim.background_location/methods. Response ID: 46

Is their any solution on this.
Thanks in Advance

**Comment by ianmuhia** [comment]
having the same issues ... any help

**Comment by H4zh4n** [comment]
@raotolehitesh, @Ianmuhia 
Did you guys find a solution ?


@BWMuller
Sorry for tagging you, but do you think this can be solved? We all seem to have the same issue.

**Comment by BWMuller** [comment]
@H4zh4n what version of the library are you on and what device details are you testing against?

Checking since I thought this one was already resolved, at least I haven't run into it recently

**Comment by H4zh4n** [comment]
@BWMuller 
- Using the package's latest version released close to a week ago, `0.16.3`.

- The device is a Xiaomi POCO X6 Pro. Running Android 15.


The app/functionalities work perfectly well until the app is closed (dragged up from the recent list). It then gives the below error, as others mentioned too :
```
W/FlutterJNI( 5668): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: com.almoullim.background_location/methods. Response ID: 528
W/FlutterJNI( 5668): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: com.almoullim.background_location/methods. Response ID: 529
.....
```


The device says that the app is still using location services as well. But the log gives the above error.
